app/views/blog/index.html.erb

```diff
@@ -1,20 +1,8 @@
-<% @section    = 'blog' %>
-<% @newest = list_posts.first %>
-<% @newest_content = preview_post(@newest[:file]) %>
-<% @page_title = "Git - #{@newest_content[1]['title']}" %>
+<% @page_title = "Git - Blog Moved" %>
 
-<% content_for :sidebar do %>
- <%= render 'list' %>
-<% end %>
+<div id="main">
+  <h1>Moved!</h1>
+
+  <p>Technical content that was previously found in the blog section of this site has been integrated into the <a href="/book">second edition of the Pro Git book</a>.
```

Comment:
doesn't rails offer a reverse router to avoid referring to links directly?

Reply:
If you use a `resource` in the router, I think it auto-generates some path helpers. I guess we do that for `book`, so maybe `book_path` exists? I dunno. It didn't seem like it was worth spending time figuring out, but if you have a working recipe I'm happy to switch to it.

Reply:
let me google and test something :)

Reply:
<p>Technical content that was previously found in the blog section of this site has been integrated into the <%= link_to('second edition of the Pro Git book', controller: 'books', action: 'show') %>.